G1ANT.Addon.Selenium/Api/Services/SeleniumDriverInstaller.cs

```diff
@@ -191,5 +200,38 @@ private static void KillProcessAndChildren(int pid)
 
             }
         }
+
+        static string DownloadChromedriver(string chromeVersion, string destinationFolder)
+        {
+            string driverPath = string.Empty;
+
+            string driverDir = Path.Combine(Path.GetTempPath(), "chromedriver");
+            if (Directory.Exists(driverDir))
+            {
+                Directory.Delete(driverDir, true);
+            }
+
+            string webDriverUrl = "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_" + chromeVersion;
+            string driverVersion = new WebClient().DownloadString(webDriverUrl);
+
+            if (!string.IsNullOrEmpty(driverVersion))
+            {
+                string driverUrl = $"https://chromedriver.storage.googleapis.com/{driverVersion}/chromedriver_win32.zip";
+
+                string zipFilePath = Path.Combine(Path.GetTempPath(), "chromedriver_win32.zip");
+                WebClient webClient = new WebClient();
+                webClient.DownloadFile(driverUrl, zipFilePath);
```

Comment:
jak pojdzie exception to chyba zablokujemy mozliwosc uzycia selenium bo za kazdym razem nam sie wywali

Reply:
No ale nie użyjemy selenium bez odpowiedniego drivera - ta funkcja wywoluje sie tylko gdy type = chrome